CodeReady Containers/DetailedStatusViewController.swift

```diff
@@ -15,19 +15,26 @@ class DetailedStatusViewController: NSViewController {
     @IBOutlet weak var diskUsage: NSTextField!
     @IBOutlet weak var cacheSize: NSTextField!
     @IBOutlet weak var cacheDirectory: NSTextField!
+    @IBOutlet weak var logs: NSTextView!
 
     let cacheDirPath: URL = userHomePath.appendingPathComponent(".crc").appendingPathComponent("cache")
 
+    var timer: Timer?
+    var font: NSFont?    = .systemFont(ofSize: 14, weight: .regular)
+
     override func viewDidLoad() {
         super.viewDidLoad()
 
         guard let appDelegate = NSApplication.shared.delegate as? AppDelegate else {
             return
         }
+
         self.updateViewWithClusterStatus(appDelegate.status)
+        self.timer = Timer.scheduledTimer(timeInterval: 2.0, target: self, selector: #selector(updateViewWithLogs), userInfo: nil, repeats: true)
```

Comment:
So this is timer based; A good first implementation.
But is there a way to look into providing long-polling from the HTTP daemon? This would be an improvement for which the command only needs to be triggered once.

Reply:
Oh yes, this is a real dumb implementation! Anything can be better than that :)

It may deserve a proper ticket for enhancement?

Reply:
Enhancement would be needed in the daemon itself; https://github.com/jcuga/golongpoll